modules/apps/expando/expando-web/src/main/java/com/liferay/expando/web/internal/portlet/ExpandoPortlet.java

```diff
@@ -268,8 +238,18 @@ private Serializable _getDefaultValue(ActionRequest actionRequest, int type)
 		}
 
 		if (type == ExpandoColumnConstants.STRING_LOCALIZED) {
-			return (Serializable)LocalizationUtil.getLocalizationMap(
-				actionRequest, "defaultValueLocalized");
+			Serializable defaultValue =
+				(Serializable)LocalizationUtil.getLocalizationMap(
+					actionRequest, "defaultValueLocalized");
+
+			if (!_isDefaultLocaleValueInformed(
+					(Map<Locale, String>)defaultValue)) {
+
+				throw new ValueDataException.MustInformDefaultLocale(
```

Comment:
@crodriguezy Since now there's only 1 usage, please remove this inner-class (inline its constructor here).

Reply:
And after removing this, I think the semantic versioning change will no longer be needed.

Reply:
@dantewang I'm not sure what you mean here, I need that specific exception to be captured in expando.jsp (also modified in this PR) in order to show the corresponding error message

Reply:
@crodriguezy I see, then please ignore this comment. Thanks!